util/buildinfo/buildinfo.go

```diff
@@ -62,8 +67,11 @@ func mergeSources(ctx context.Context, buildSources map[string]string, frontendS
 		switch sourceID := src.(type) {
 		case *source.ImageIdentifier:
 			for i, fsrc := range frontendSources {
+				if fsrc.Type != binfotypes.SourceTypeDockerImage {
```

Comment:
What are these new cases?

Reply:
This is the same as previous, just reduce code indent.